blog/2024-10-30-openrefine-developer-role.md

```diff
@@ -0,0 +1,63 @@
+* Total funding available for the position: USD 90,000 annually 
+
+## Role Summary
+
+This part-time role for an OpenRefine Developer, contracted through [Code for Science & Society Inc.](https://www.codeforsociety.org/) (a 501(c)(3) charitable organization in the USA), will focus on enhancing OpenRefine's stability, reliability, and quality to support the project’s long-term sustainability. This role is funded by the [EOSS-5 grant](/funding#2022-eoss-5).
+
+Key responsibilities include bug fixes, quality improvements, and ongoing maintenance, with an emphasis on features supported by grant funding. Over time, the developer will expand into release management, security maintenance, and mentorship within internship programs (e.g., GSoC, Outreachy).
+
+## Core Responsibilities
+* **Bug Fixes & Quality Improvements**
+  * Prioritize and address bugs within the codebase to enhance stability and performance.
+  * Improve reproducibility features (per EOSS-5 grant requirements).
+  * Implement quality improvements through refactoring, optimization, and code maintenance.
+* **Community Support & Engagement**
+  * Participate in community forums to support OpenRefine users and volunteers, fostering a positive and engaged user community.
+* **Pull Request Reviews & Ticket Triage**
+  * Once familiar with the codebase, actively engage with community-submitted pull requests, providing constructive feedback and participating in collaborative review processes.
+  * Assist with ticket triage to identify and prioritize key issues and opportunities.
+* **Progressive Onboarding into Release Management & Security Updates**
+  * Upon receiving commit privileges, assist in merging pull requests to support the workflow of community contributions.
+  * Take on Release Manager responsibilities to prepare and publish new releases, coordinating with the community on release planning.
+  * With further onboarding, address security advisories and implement required updates and patches to maintain OpenRefine’s security.
+* **Internship Program Participation**
+  * Support programs such as Google Summer of Code (GSoC) and Outreachy, mentoring interns, reviewing their contributions, and fostering a supportive learning environment.
+
+## Work Environment & Reporting Structure
+
+This is a fully remote role with flexible working hours, allowing the developer to organize their time freely over the month. This role requires a self-motivated, independent approach, with minimal direct supervision. The OpenRefine Developer will primarily work with volunteers, including members of the Advisory Committee and most technical contributors. The OpenRefine community collaborates asynchronously through public forums and GitHub, accommodating the diverse time zones and work patterns of contributors. Occasional travel may be expected to attend relevant conferences and meetings, enabling valuable in-person engagement with the OpenRefine community and the broader open-source ecosystem.
```

Comment:
```suggestion
This is a fully remote role with flexible working hours, allowing the developer to organize their time freely over each month. This role requires a self-motivated, independent approach, with minimal direct supervision. The OpenRefine Developer will primarily work with volunteers, including members of the Advisory Committee and most technical contributors. The OpenRefine community collaborates asynchronously through public forums and GitHub, accommodating the diverse time zones and work patterns of contributors. Occasional travel may be expected to attend relevant conferences and meetings, enabling valuable in-person engagement with the OpenRefine community and the broader open-source ecosystem.
```

Reply:
Shouldn't this be each month, since it's a 9 month contract and not 1 month contract?